src/main/java/com/tiff/tffnserachservice/controller/TiffnerController.java

```diff
@@ -0,0 +1,80 @@
+package com.tiff.tffnserachservice.controller;
+
+
+import com.tiff.tffnserachservice.dto.TiffnerDTO;
+import com.tiff.tffnserachservice.exception.TiffnNotFoundException;
+import com.tiff.tffnserachservice.model.Tiffner;
+import com.tiff.tffnserachservice.repository.TiffnerRepository;
+import com.tiff.tffnserachservice.service.TiffnerService;
+import lombok.extern.slf4j.Slf4j;
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.BeanUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.text.ParseException;
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/tffn-search")
+@Slf4j
+public class TiffnerController {
+
+    @Autowired
+    private TiffnerRepository tiffnerRepository;
+
+    @Autowired
+    private TiffnerService tiffnerService;
+
+    @GetMapping("/tffns")
+    public List<Tiffner> findTiffns() {
+        return tiffnerRepository.findAll();
+    }
+
+    @GetMapping("/tffns/{id}")
+    public ResponseEntity<Tiffner> findTffn(@PathVariable Long id) {
+        return tiffnerRepository.findByTiffnerId(id).map(ResponseEntity::ok).orElseThrow(() -> new TiffnNotFoundException(id));
+    }
+
+
+    @DeleteMapping("/tffns/{id}")
+    public ResponseEntity<Tiffner> deleteTffn(@PathVariable Long id) {
+        Optional<Tiffner> tiffner = Optional.ofNullable(tiffnerRepository.findByTiffnerId(id).orElseThrow(() -> new TiffnNotFoundException(id)));
+
+        if (tiffner.isPresent()) {
+            tiffnerRepository.delete(tiffner.get());
+            log.info(tiffner.toString() + " was deleted");
```

Comment:
make this debug level. we dont want to log each request/response. only when we set the application to debug level should we see this, the errors one are good. You should use this format for log -> log.info("tiffner {} was deleted", tiffner.toString);